.github/workflows/e2e.yml

```
@@ -1,9 +1,15 @@
 name: Playwright Tests
+
 on:
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)